MWContentDisplayPlugin.podspec

```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |s|
     s.name                  = 'MWContentDisplayPlugin'
-    s.version               = '1.7.0'
+    s.version               = '1.7.1'
```

Comment:
would it make more sense to move to `1.8.0` as it adds a feature?

Reply:
It does. I bumped it to 1.8.0